LinuxFunctionsStacks.json

```diff
@@ -146,7 +146,7 @@
                             "~2",
                             "~3"
                         ],
-                        "isDefault": true,
+                        "isDefault": false,
```

Comment:
I actually think we need to revert this. Since `3.6` supports extension version `~2` and `~3`, it should still be `"isDefault": true,` because it's the `~2` default, right? That's why dotnet and node have multiple defaults.

I think both `3.6` and `3.8` need to be marked as default, `3.6` for `~2` and `3.8` for `~3`. Unless we're adding `~2` support for `~3.8`?

(this is also why the `isDefault` property is a little wonky)

Reply:
> I think both 3.6 and 3.8 need to be marked as default, 3.6 for ~2 and 3.8 for ~3. Unless we're adding ~2 support for ~3.8?
> (this is also why the isDefault property is a little wonky)

So currently 3.7 and 3.8 are marked as default - does this mean that when someone uses ~3, it'll choose 3.8 and ~2 will take 3.7 as default?

Reply:
that is correct!